src/components/Pagination/Pagination.tsx

```diff
@@ -62,35 +83,39 @@ const Wrapper = styled.div`
   display: inline-block;
 `
 const List = styled.ul`
-  ${({ theme }: InjectedProps) => {
-    const { frame, palette } = theme
-
-    return css`
-      display: flex;
-      border-radius: ${frame.border.radius.s};
-      border: 1px solid ${palette.BORDER};
-      margin: 0;
-      padding: 0;
-
-      & > li {
-        margin: 0;
-        padding: 0;
-        list-style: none;
-
-        &:not(:first-child) {
-          border-left: ${frame.border.default};
-        }
-
-        &:first-child > .PaginationItem {
-          border-top-left-radius: ${frame.border.radius.s};
-          border-bottom-left-radius: ${frame.border.radius.s};
-        }
-
-        &:last-child > .PaginationItem {
-          border-top-right-radius: ${frame.border.radius.s};
-          border-bottom-right-radius: ${frame.border.radius.s};
-        }
+  display: flex;
+  margin: 0;
+  padding: 0;
+  > li {
+    list-style: none;
+    margin: 0 4px;
```

Comment:
There are unnecessary space that is not defined.
![image](https://user-images.githubusercontent.com/1614892/66383899-c652ff00-e9f8-11e9-9a2d-b1b0b31934d8.png)

Please set `letter-spacing` props to delete unnecessary space.
https://qiita.com/BEMU/items/0f500e0306471073ed63

Reply:
It was taking the space like `4 + 4 = 8`.
I changed how to apply margin in the commit below, though.
https://github.com/kufu/smarthr-ui/commit/eea395a59f830512b4273d45d461914341e18755